config.ml

```diff
@@ -10,8 +12,10 @@ let ipv4_only = Runtime_arg.ipv4_only ~group:"sys-net" ()
 let ipv6_only = Runtime_arg.ipv4_only ~group:"sys-net" ()
 let stack = direct_stackv4v6 ~ipv4_only ~ipv6_only default_network ethernet arp ipv4 ipv6
 
+let config_fn = runtime_arg ~pos:__POS__ "Unikernel.config_fn"
```

Comment:
I prefer `config_key` instead of `config_fn`. Otherwise, good to merge.

Reply:
Thank you @dinosaure , I hope [401d5e2](https://github.com/robur-coop/qubes-miragevpn/pull/4/commits/401d5e2af6e903b45a9807877b2a191b55f92482) is in a good shape now :)